dashboard.py

```diff
@@ -242,9 +242,13 @@ def make_ab_confidence_map(
             "runtime",
         ]
     ]
+
     runtimes = df["runtime"].unique()
+    if len(runtimes) < 2 or baseline not in runtimes:
```

Comment:
I've made a similar fix in #432. But will this actually solve the problem? I think the problem that I saw was when `runtimes` was entirely full of `baseline`, not if `baseline` didn't appear.

Reply:
I now moved this fix to #434 and it does solve the problem